pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -206,6 +211,10 @@ func (r *Reconciler) checkDeployments(manifest *mf.Manifest, instance *servingv1
 			}
 		}
 	}
+	if instance.Status.IsDeploying() {
```

Comment:
Mmh, what does IsDeploying actually check?

To me this if block reads as "if still deploying, report deployment is read", which seems preemptive. It's possible that IsDeploying()'s implementation is really IsDeployed() though.

Reply:
`IsDeploying()` returns true when the manifest install succeeded but the deployment is not ready. Adding this block here is to capture the latency time before marking the deployment as ready in the next line.